lib/node_modules/@stdlib/repl/README.md

```diff
@@ -176,6 +199,126 @@ repl.clearUserDocs();
 repl.close();
 ```
 
+#### Repl.prototype.themes()
+
+Returns all available themes in the syntax-highlighter.
+
+```javascript
+var debug = require( '@stdlib/streams/node/debug-sink' );
+
+// Create a new REPL:
+var repl = new REPL({
+    'output': debug()
+});
+
+// ...
+
+// Fetch all available themes:
+repl.themes();
+
+// ...
+
+// Close the REPL:
+repl.close();
+```
+
+#### Repl.prototype.getTheme()
+
+Returns a theme's color palette from the syntax-highlighter.
+
+```javascript
+var debug = require( '@stdlib/streams/node/debug-sink' );
+
+// Create a new REPL:
+var repl = new REPL({
+    'output': debug()
+});
+
+// ...
+
+// Fetch all available themes:
+repl.themes();
+
+// ...
+
+// Close the REPL:
+repl.close();
+```
+
+#### Repl.prototype.addTheme()
+
+Adds a color theme to the syntax-highlighter.
+
+```javascript
+var debug = require( '@stdlib/streams/node/debug-sink' );
+
+// Create a new REPL:
+var repl = new REPL({
+    'output': debug()
+});
+
+// ...
+
+// Add a user-defined theme:
+repl.addTheme( 'myTheme', {
+    'keyword': 'red',
+    'variable': 'green'
+
+    // ...
+});
+
+// ...
+
+// Close the REPL:
+repl.close();
+```
+
+#### Repl.prototype.deleteTheme()
+
+Deletes a specified theme from the syntax-highlighter.
+
+```javascript
+var debug = require( '@stdlib/streams/node/debug-sink' );
+
+// Create a new REPL:
+var repl = new REPL({
+    'output': debug()
+});
+
+// ...
```

Comment:
This example is incomplete. First add the theme `myTheme` and then later on delete it. Otherwise, `myTheme` lacks context.

Reply:
I would also expect `deleteTheme` to error if provided a non-existent theme name.